packages/scm/src/browser/style/index.css

```diff
@@ -190,8 +198,6 @@
 }
 
 .theia-scm .scmItem:hover {
-    background-color: var(--theia-list-hoverBackground);
```

Comment:
Why did we remove VS Code theming? Did you check that VS Code don't apply it? The same for `scm-theia-header:hover`

Reply:
This is necessary because otherwise the theming from the original source control view messes with the tree selection and looks awful.
![Screenshot from 2020-04-28 15-35-41](https://user-images.githubusercontent.com/87310/80549360-ebb75280-89b4-11ea-804f-9afabadf064d.png)

Reply:
Could you file a follow-up issue? We need to double check how it should be applied and do it. But I don't want to hinder this PR with it.